server/errorHandler.js

```diff
@@ -9,6 +9,14 @@ exports.notFound = (req, res, next) => {
   next(error)
 }
 
+exports.authError = (error, req, res, next) => {
+  if ([401,403]
```

Comment:
I believe it will wire a 403 on expiry and the 401 is caught by the express auth firmware earlier on in the middleware (authorisation.js) and hence 401 is impossible, however I'm only 99.9% and can't test it until I see what auth does tomorrow (jwt expiry is now 12hrs). Technically it's assuming only the CCS can produce 40X, unsure if that's exactly the case being we use user-preference-service. Good enough for now.